code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/FedotovAndreiTest.java

```diff
@@ -0,0 +1,144 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class FedotovAndreiTest {
+
+    private final double delta = 0.001;
+
+    @Test
+    public void canGiveImagineNumber() {
+        double imagine = 0.0;
+        ComplexNumber z1 = new ComplexNumber(1, imagine);
+        assertEquals(imagine, z1.getImaginary(), delta);
+    }
+
+    @Test
+    public void canGiveRealNumber() {
+        double real = 0.0;
+        ComplexNumber z1 = new ComplexNumber(real, 1);
+        assertEquals(real, z1.getReal(), delta);
+    }
+
+    @Test
+    public void canGiveHashCode() {
+        ComplexNumber z1 = new ComplexNumber(0, 0);
+        assertTrue((z1.hashCode() >= 0));
```

Comment:
Is there an assertion like assertEqualsOrGreater?

Reply:
No just validation that we receive a value rather than null.